data/SAMPLE-cat.html

```diff
@@ -1075,7 +1016,7 @@
    
 </section><a class="anchor" name=""></a><section>
    <h1>Association classes </h1>
-   <a class="anchor" name="global_d6505e4874"></a><div>
+   <a class="anchor" name="global_d6505e4874"></a><section class="notoc">
```

Comment:
Dit moet niet een `<div>` zijn maar `<section>` en als die niet mag voorkomen in de inhoudsopgave, dan met `class="notoc"`.